modules/grpc/bigquery/bigquery-dest.cpp

```diff
@@ -113,7 +150,20 @@ DestinationDriver::init()
   GlobalConfig *cfg = log_pipe_get_config(&this->super->super.super.super.super);
   log_template_options_init(&this->template_options, cfg);
 
-  this->construct_schema_prototype();
+  if (this->protobuf_schema.proto_path.empty())
+    this->construct_schema_prototype();
+  else
+    {
+      if (!this->load_protobuf_schema())
+        return false;
```

Comment:
Can we make the fields permanent somehow after a successful load, so we can revert to it in case of a failed reload (e.g. syntax error is introduced in the `.proto` file)?